screens/teachers/TeacherProfileScreen.js

```diff
@@ -4,105 +4,114 @@ import { styles } from '../../styles/styles';
 import { colors } from '../../styles/colors';
 import { textStyles } from '../../styles/textStyles';
 import { teacherStyles } from '../../styles/teacherStyles';
+import { commonStyles } from '../../styles/styles';
 import { getRequest } from '../../lib/requests';
 import { APIRoutes } from '../../config/routes';
 
 class TeacherProfileScreen extends React.Component {
-  static navigationOptions = ({navigation}) => ({
-    // const { navigate } = this.props.navigation;
-      headerRight: 
-        <Button
-           title="Edit Profile" onPress={()=>{ navigation.navigate('EditTeacherProfile'); }}  
-        />
-
-  });
 
   constructor(props) {
     super(props);
-    this._fetchTeachers = this._fetchTeachers.bind(this);
-    this._renderTeachers = this._renderTeachers.bind(this);
+    this._fetchTeacher = this._fetchTeacher.bind(this);
+    this._renderTeacher = this._renderTeacher.bind(this);
     this.state = {
-      teachers : { },
+      teacher : { },
       isLoading : true,
     }
   }
 
   componentDidMount() {
-    this._fetchTeachers();
+    this._fetchTeacher();
   }
 
-  _fetchTeachers() {
+  _fetchTeacher() {
     const successFunc = (responseData) => {
-      this.setState({ teachers: responseData, isLoading: false });
+      this.setState({ teacher: responseData, isLoading: false });
+
     }
     const errorFunc = (error) => {
       console.error(error);
     }
-    getRequest(APIRoutes.getTeachersPath(), successFunc, errorFunc);
+    getRequest(APIRoutes.getTeacherPath(1), successFunc, errorFunc);
+
   }
 
-  _renderTeachers() {
-    return this.state.teachers.map(function(teacher, i) {
-      return(
-        <View key={i} style={teacherStyles.base}>
-          <View style={teacherStyles.div_1}>
-            <View style={teacherStyles.div_2}>
-            <Text style={textStyles.titleLarge}>
-            {teacher.first_name} {teacher.last_name}
-            </Text>
-            </View>
-
-            <View style={teacherStyles.div_2}>
-            <Text style={textStyles.titleSmall}>
-            Dream ID
-            </Text>
-            <Text style={textStyles.body}>
-            {teacher.dream_id}
-            </Text>
-            </View>
-
-            <View style={teacherStyles.div_2}>
-            <Text style={textStyles.titleSmall}>
-            Email
-            </Text>
-            <Text style={textStyles.body}>
-            {teacher.email}
-            </Text>
-            </View>
-
-            <View style={teacherStyles.div_2}>
-            <Text style={textStyles.titleSmall}>
-            Phone Number
-            </Text>
-            <Text style={textStyles.body}>
-            {teacher.phone}
-            </Text>
-            </View>
+  _renderTeacher() {
+    return(
+      <View style={teacherStyles.base}>
+        <View style={teacherStyles.div_1}>
+          <View style={teacherStyles.div_2}>
+          <Text style={textStyles.titleLarge}>
```

Comment:
Nit: Nest these text views. Also I feel like this can be modularized in the future